src/App.js

```diff
@@ -1333,7 +1335,76 @@ render() {
             )
           }
 
+
           switch(view) {
+            case 'yourmodule':
```

Comment:
I'd recommend renaming the route from `yourmodule` to something more descriptive.
Additionally, can we put this `case` below the `main` view case? I don't think we need the `if-else` statement either.